cockroachcloud/backups-page.md

```diff
@@ -37,14 +37,14 @@ A list of your full cluster backups displays on your cluster's **Backups** page.
 For each backup, the following details display:
 
 <div class="filter-content" markdown="1" data-scope="dedicated">
-- The date and time the backup was taken (**Data From**)
-- The **Status** of the backup
+- The date and time the backup was taken (**Data From**) 
+- The **Status** of the backup 
```

Comment:
The 'Status' field will be removed pretty soon (over the coming month). Just FYI. 

We may want to file a docs issue to update docs after these two epics are done - https://cockroachlabs.atlassian.net/browse/CRDB-19573 
https://cockroachlabs.atlassian.net/browse/CRDB-19836

Reply:
filed issue here https://cockroachlabs.atlassian.net/browse/DOC-6255